easier68k-cli/test.x68

```diff
@@ -0,0 +1,15 @@
+* Equates section
```

Comment:
unsure if there's a good way to do automated tests on Cmd. I think maybe we should stick this test code somewhere else. (or under a 'test' dir)